hook_pre-commit.go

```diff
@@ -14,16 +14,16 @@ func HookPreCommitDisable(args []string) error {
 
 func HookPreCommitRun(args []string) error {
 	options := map[string]interface{}{
-		"commit": false,
-		"staged": true,
+		"commit-files": false,
+		"staged-files": true,
```

Comment:
This change is because the original author had added the suffix `-files` but didn't update throughout the project. 👍 good catch @alain-hoang